src/datagen/main/java/mekanism/client/lang/MekanismLangProvider.java

```diff
@@ -378,14 +378,15 @@ private void addPigments() {
 
     private void addSlurries() {
         add(MekanismAPI.EMPTY_SLURRY, "Empty");
-        for (Map.Entry<PrimaryResource, SlurryRegistryObject<Slurry, Slurry>> entry : MekanismSlurries.PROCESSED_RESOURCES.entrySet()) {
+        for (Map.Entry<PrimaryResource, SlurryRegistryObject<Slurry, Slurry, Slurry>> entry : MekanismSlurries.PROCESSED_RESOURCES.entrySet()) {
             addSlurry(entry.getValue(), formatAndCapitalize(entry.getKey().getName()));
         }
     }
 
-    private void addSlurry(SlurryRegistryObject<Slurry, Slurry> slurryRO, String name) {
+    private void addSlurry(SlurryRegistryObject<Slurry, Slurry, Slurry> slurryRO, String name) {
         add(slurryRO.getDirtySlurry(), "Dirty " + name + " Slurry");
         add(slurryRO.getCleanSlurry(), "Clean " + name + " Slurry");
+        add(slurryRO.getPureSlurry(), "Pure " + name + " Slurry");
```

Comment:
I will probably review this PR in more detail sometime later this week (and figure out by then if we want to expand Mek to including 6x ore processing in general), but one thing that probably should be changed is instead of `Pure x Slurry` it should be `Purified x Slurry`

Reply:
ok

Reply:
done